test/BUILD.gn

```diff
@@ -91,6 +91,8 @@ test("brave_unit_tests") {
     "//chrome/test:disable_thinlto_cache_flags",
   ]
 
+  defines = [ "ALLOW_IMPLICIT_ATOMIC_STRING_CONVERSIONS" ]
```

Comment:
we're setting this for all unit tests??

Reply:
cc: @cdesouza-chromium ☝️ It looks like `third_party/blink/renderer/brave_font_whitelist_unittest.cc` is the only one that needs it, but it's in the same target with all other unit tests in `test:brave_unit_tests`.

Reply:
Fixed in b76c84feaf8c93bd93d41b95609c88f201937983